tests/test_requests.py

```diff
@@ -844,7 +843,7 @@ def test_conflicting_post_params(self, httpbin):
             with pytest.raises(ValueError):
                 requests.post(url, data='[{"some": "data"}]', files={'some': f})
             with pytest.raises(ValueError):
-                requests.post(url, data=u('[{"some": "data"}]'), files={'some': f})
+                requests.post(url, data=u'[{"some": "data"}]', files={'some': f})
```

Comment:
Do we need `u'...'`?

Reply:
No we don't, I have a separate linting PR that removes these lines entirely since they're redundant. I can move that into this one.

Reply:
`pyupgrade **/*.py --py37-plus` will deal with a bunch of `u''`s and other stuff.